bot/cogs/information.py

```diff
@@ -104,36 +100,18 @@ def __init__(self, bot: Bot):
         features = ", ".join(ctx.guild.features)
         region = ctx.guild.region
 
-        # How many of each type of channel?
         roles = len(ctx.guild.roles)
-        channels = ctx.guild.channels
-        text_channels = 0
-        category_channels = 0
-        voice_channels = 0
-        for channel in channels:
-            if type(channel) == TextChannel:
-                text_channels += 1
-            elif type(channel) == CategoryChannel:
-                category_channels += 1
-            elif type(channel) == VoiceChannel:
-                voice_channels += 1
+        member_count = ctx.guild.member_count
+
+        # How many of each type of channel?
+        channels = Counter({TextChannel: 0, VoiceChannel: 0, CategoryChannel: 0})
+        for channel in ctx.guild.channels:
+            channels[channel.__class__] += 1
 
         # How many of each user status?
-        member_count = ctx.guild.member_count
-        members = ctx.guild.members
-        online = 0
-        dnd = 0
-        idle = 0
-        offline = 0
-        for member in members:
-            if str(member.status) == "online":
-                online += 1
-            elif str(member.status) == "offline":
-                offline += 1
-            elif str(member.status) == "idle":
-                idle += 1
-            elif str(member.status) == "dnd":
-                dnd += 1
+        statuses = Counter({status.value: 0 for status in Status})
+        for member in ctx.guild.members:
+            statuses[member.status.value] += 1
```

Comment:
It's better to use the enum itself rather than its value. Strings are more prone to programmer errors.

Reply:
Changed, thank you for the feedback :)